mitmproxy/proxy/layers/http/_http_h2.py

```diff
@@ -147,6 +157,8 @@ def stream_window_updated(self, stream_id: int) -> bool:
             available_window -= len(chunk.data)
             if not self.stream_buffers[stream_id]:
                 del self.stream_buffers[stream_id]
+                if self.stream_trailers.get(stream_id):
```

Comment:
```suggestion
                if stream_id in self.stream_trailers:
```
(We may have an empty trailers frame, which would otherwise evaluate to False)